Tests/demistomock/demistomock.py

```diff
@@ -1044,3 +1044,49 @@ def getModules():
 
 def get_incidents():
     return {}
+
+
+def internalHttpRequest(method, uri, body=None):
+    """Run an internal HTTP request to the XSOAR server. The request runs with the permissions of the 
+    executing user, when a command is being executed manually (such as via the warroom or when browsing a widget).
+    When run via a playbook, will run with a readonly user with limited permissions isolated to the current incident only.
+    Available for both Integrations and Scripts starting from Server version 6.1.
+
+    Args:
+        method (str): HTTP method such as: GET or POST
+        uri (str): Server uri to request. For example: "/contentpacks/marketplace/HelloWorld".
+        body (str, optional): Optional body for a POST request. Defaults to None.
```

Comment:
for all other functions, the format for default value is `(Default value = None)` so i think better to change that or change others to stay consistent